Expand on reason why VMs would not show up on list

andynava opened the pull request:
If a VM is already being backed up for the current or another Recovery Services vault, it will not be shown in the list, so point this out.

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

andynava committed:
Expand on reason why VM would not show up on list

If a VM is already being backed up for the current or another Recovery Services vault, it will not be shown in the list, so point this out.

andynava committed:
Update backup-azure-vms-first-look-arm.md
